File: WikiPhytoScrapper/WikiPhytoScrapper/Models/PlantFamily.cs
using System.Collections.Generic;

namespace WikiPhytoScrapper.Models
{
    public class PlantFamily
    {
        public string Id { get; set; }
        public string Name { get; set; }
        public List<Plant> Plants { get; set; }
        public string Link { get; set; }
    }
}

File: Website/src/models/plant-family.ts
import PlantDetail from "./plant-detail";

export default class PlantFamily {
    public id: string;
    public name: string;
    public link: string;
    public plants: PlantDetail[];
    constructor(id: string, name: string, link: string) {
        this.id = id;
        this.name = name;
        this.link = link;
        this.plants = [];
    }
}